lib/user/widgets/user_selector.dart

```diff
@@ -30,6 +30,10 @@ class UserSelector extends StatefulWidget {
   final String? parentCommentActorId;
   final void Function(CommentView)? onParentCommentChanged;
 
+  /// Whether the user is allowed to change the active account
+  /// (e.g., it should not be allowed during edit)
+  final bool allowChange;

```

Comment:
Just one nitpick on the naming of this: what do you think of `enableAccountSwitching`?

Reply:
Done!